docs/SUMMARY.md

```diff
@@ -41,62 +41,8 @@
 
 ## References
 
-<!-- auto-created, do not edit, begin -->
```

Comment:
this is not good !!

Reply:
yes, actually it is on purpose: the API reference TOC was moved to references/api/README.md -> that one is auto generated

Reply:
aah ok I see, thanks !